Observer/OrderPlacedHandler.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Copyright © TrueLayer Ltd. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace TrueLayer\Connect\Observer;
+
+use Magento\Framework\Event\Observer;
+use Magento\Framework\Event\ObserverInterface;
+use Magento\Sales\Api\OrderRepositoryInterface;
+use Magento\Sales\Model\Order;
+use TrueLayer\Connect\Api\Log\RepositoryInterface as LogRepository;
+
+
+class OrderPlacedHandler implements ObserverInterface
```

Comment:
We use this observer to ensure that orders that are placed are placed in the PENDING PAYMENT state straight away.